lib/ReactViews/Custom/Chart/BottomDockChart.tsx

```diff
@@ -147,56 +157,53 @@ class Chart extends React.Component {
   get initialScales() {
     return this.chartItems.map((c) => ({
       x: this.initialXScale,
-      y: this.yAxes.find((y) => y.units === c.units).scale
+      y: this.yAxes.find((y) => y.units === c.units)?.scale
     }));
   }
 
   @computed
   get zoomedScales() {
     return this.chartItems.map((c) => ({
       x: this.xScale,
-      y: this.yAxes.find((y) => y.units === c.units).scale
+      y: this.yAxes.find((y) => y.units === c.units)?.scale
     }));
   }
 
   @computed
   get cursorX() {
     if (this.pointsNearMouse.length > 0)
+      // @ts-expect-error xScale is not a function.
       return this.xScale(this.pointsNearMouse[0].point.x);
     return this.mouseCoords && this.mouseCoords.x;
   }
 
   @computed
   get pointsNearMouse() {
-    if (!this.mouseCoords) return [];
+    const coords = this.mouseCoords;
+    if (!isDefined(coords)) return [];
```

Comment:
I assumed 'this.mouseCoords` is not updated during the time this function runs, so the `undefined` part of the type could be eliminated before doing the map over the chartItems.